specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -904,6 +1058,45 @@
         }
       }
     },
+    "IscsiTargetUpdateProperties": {
+      "type": "object",
+      "description": "Properties of iSCSI Target request payload.",
+      "required": [
+        "tpgs"
+      ],
+      "properties": {
+        "provisioningState": {
+          "description": "State of the operation on the resource",
+          "type": "string",
+          "$ref": "#/definitions/ProvisioningState",
+          "readOnly": true
+        },
+        "status": {
+          "type": "string",
+          "enum": [
+            "Invalid",
+            "Unknown",
+            "Healthy",
+            "Unhealthy"
+          ],
+          "x-ms-enum": {
+            "name": "OperationalStatus",
+            "modelAsString": true
+          },
+          "readOnly": true,
```

Comment:
Per my understanding, this is for update IscsiTarget input, so there should be no readonly property here (as it's only for input).